.github/workflows/ci.yml

```diff
@@ -205,7 +205,7 @@ jobs:
         set -ex
         cd contrib/deploy-onefuzz-via-azure-devops
         python -m pip install --upgrade pip
-        python -m pip install tox pipenv
+        python -m pip install tox pipenv==2022.11.11
```

Comment:
Can you create a work item to un-pin this once the root cause is fixed?

Reply:
https://github.com/microsoft/onefuzz/issues/2651